backend/db/refresh.py

```diff
@@ -36,33 +36,48 @@ def refresh_db(
     time will not be fetched, resulting in an incomplete database. Therefore by default this will raise an error unless
     this is set to True."""
     local = use_local_database
-    print('INFO: Starting database refresh.', flush=True)  # flush: for gh action
     t0, t0_str = datetime.now(), current_datetime()
-    update_db_status_var('last_refresh_request', t0_str, local)
-    update_db_status_var('refresh_status', 'active', local)
-    try:
-        with get_db_connection(local=local) as con:
-            last_refresh = last_refresh_timestamp(con)
-            if since and dp.parse(since) > dp.parse(last_refresh) and not force_non_contiguity:
-                raise ValueError(SINCE_ERR)
-            since = since if since else last_refresh
 
+    with get_db_connection(local=local) as con:
+        last_refresh = last_refresh_timestamp(con)
+        if since and dp.parse(since) > dp.parse(last_refresh) and not force_non_contiguity:
+            raise ValueError(SINCE_ERR)
+        since = since if since else last_refresh
+
```

Comment:
I moved this entire part of the code up, because when working with Siggie it seemed like making as few connections as possible in a function was a priority for them. On running in debugger it seemed like this change didn't affect the program's functionality.

Reply:
Probably not a bad idea.